src/components/FormikMovieForms/FormikField.tsx

```diff
@@ -1,7 +1,14 @@
 import React from "react";
 import { Field } from "formik";
 
-const FormikField = ({ title, path, myErrors, myTouched }) => {
+interface FieldInterface {
+  title: string;
+  path: string;
+  myErrors: any;
+  myTouched: any;
+}
+
+const FormikField = ({ title, path, myErrors, myTouched }: FieldInterface) => {
```

Comment:
```suggestion
const FormikField: React.FC = ({ title, path, myErrors, myTouched }: FieldInterface) => {
```

I guess you got the idea :D
Check if you've typed other components